client/state/sites/selectors.js

```diff
@@ -503,6 +503,10 @@ export function getSitePlan( state, siteId ) {
 	return site.plan;
 }
 
+export function getSitePlanSlug( state, siteId ) {
```

Comment:
We should eventually move these over into global selectors, but let's hold off on that for now.